src/gam/__init__.py

```diff
@@ -8262,6 +8280,10 @@ def doUpdateUser(users, i):
                 'primary': False,
                 'address': user_primary
             }]
+        if (verifyNotInvitable and'primaryEmail' in body and
+            gapi_cloudidentity_userinvitations.get_is_invitable_user(body['primaryEmail'])):
+            sys.stderr.write(f'User {user} not updated, new primaryEmail {body["primaryEmail"]} is an unmanaged account\n')
```

Comment:
should be calling controlflow.system_error_exit() here instead of sys.exit directly.